lib/quby/questionnaires/dsl/questionnaire_builder.rb

```diff
@@ -54,6 +54,17 @@ def sbg_key(sbg_key)
           @questionnaire.sbg_key = sbg_key
         end
 
+        def sbg_domain(sbg_key, from: nil, till: nil, outcome: nil, key: nil, primary: false)
```

Comment:
de key is maar in 1 geval anders dan de standaard sbg_key die we mee geven van een lijst (een oq45 dacht ik) in alle andere gevallen zou je het met een default niet elke keer op hoeven geven. 
En gezien het optioneel is hoef je dan ook niet uit te leggen dan gebruikers moeten defaulten naar de sbg_key van de lijst.

Moet de outcome niet verplicht zijn?